.github/workflows/continuous-delivery.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3.5.1
         with:
-          node-version: 14
+          node-version: 16
```

Comment:
nit: FYI this action can be used with an .nvmrc file, then in future updates it only needs to be updated there, for [example](https://github.com/ridedott/portal/blob/29197c6e9098146481924e72eb268af7496c08e5/.github/workflows/continuous-integration.yaml#L34)

Reply:
Interesting, I didn't know that.  Will change.

Reply:
Done, thanks @aaneitchik ! :D